frontend/packages/ceph-storage-plugin/src/components/independent-dashboard-page/utilization-card/card.tsx

```diff
@@ -0,0 +1,50 @@
+import * as React from 'react';
+import DashboardCardTitle from '@console/shared/src/components/dashboard/dashboard-card/DashboardCardTitle';
+import DashboardCardHeader from '@console/shared/src/components/dashboard/dashboard-card/DashboardCardHeader';
+import DashboardCard from '@console/shared/src/components/dashboard/dashboard-card/DashboardCard';
+import { Dropdown, humanizeBinaryBytes } from '@console/internal/components/utils';
+import UtilizationBody from '@console/shared/src/components/dashboard/utilization-card/UtilizationBody';
+import { ByteDataTypes } from '@console/shared/src/graph-helper/data-utils';
+import DashboardCardBody from '@console/shared/src/components/dashboard/dashboard-card/DashboardCardBody';
+import {
+  useMetricDuration,
+  Duration,
+} from '@console/shared/src/components/dashboard/duration-hook';
+import { PrometheusUtilizationItem } from '@console/internal/components/dashboard/dashboards-page/overview-dashboard/utilization-card';
+import { StorageDashboardQuery, UTILIZATION_QUERY } from '../../../constants/queries';
+
+const UtilizationCard: React.FC = () => {
+  const [duration, setDuration] = useMetricDuration();
+  const [timestamps, setTimestamps] = React.useState<Date[]>();
+
+  return (
+    <DashboardCard>
+      <DashboardCardHeader>
+        <DashboardCardTitle>Utilization</DashboardCardTitle>
+        <Dropdown items={Duration} onChange={setDuration} selectedKey={duration} title={duration} />
```

Comment:
Just curious, `Duration` is an enum type but we're passing it as value via `items` prop?

I see we do this on other places too, so I guess it works :smiley: